README.md

```diff
@@ -13,14 +13,35 @@ a great set of Pandoc-like extensions and plenty of extra goodies.
 
 ## Installation
 
+### Quickstart
+
 This module collection requires SILE v0.14 or upper.
 
 Installation relies on the **luarocks** package manager.
 
+To get your Lua version for SILE:
```

Comment:
I have 11 SILE modules already on luarocks (https://luarocks.org/modules/Omikhleia), soon 12... I think it would be best if the necessary instructions were made clear enough in the SILE manual -- the section on how to install 3rd-party packages could be improved -- so to only have to refer to it, eventually.

Also, though I like this new luarocks way of installing package a lot, such small shell scripts are bit frightening. I am actually unsure the Lua versioning thing is really necessary (I have both Lua 5.2 and 5.4 on my system until I am done migrating my older stuff, but most users would only need one). Again, the SILE distribution, IMHO, should alleviate the users and package designers' task by providing a wrapper, if so many subtle things have to be done. Installation should always be a one-liner for any good distribution system.

Reply:
It's necessary. Plenty of packages install their own Lua versions, I've seen systems with 6 Lua's before, and the person who owned it didn't even know Lua lol. Getting the `sile` script, taking its shebang line, then calling that binary with `-v`, is the only reliable way I've found.

If you try to install it with `--lua-version 5.4` and your `sile` runs as `5.2` then it won't work.

Reply:
Arf, right. I wish we could do something such as `sile --install xxx.sile` and let ensure the proper magic is done...